src/enter.js

```diff
@@ -6,25 +6,29 @@ var Promise = require('bluebird');
 
 module.exports = function(ipp, commander) {
   return new Promise(function(resolve, reject) {
-    var logger = commander.quiet ? {log: _.noop, error: _.noop} : console;
-    var maxRequests = commander.max ? parseInt(commander.max, 10) : 100;
+    var logger = {log: _.noop, error: _.noop}; // no crawler logging for now
```

Comment:
I need to revisit logging in the crawler itself.
I'm not sure how useful it is.

This would be a separate PR